dask_image/ndfilters/_conv.py

```diff
@@ -17,7 +17,7 @@ def convolve(image,
     depth, boundary = _utils._get_depth_boundary(image.ndim, depth, "none")
```

Comment:
I haven't yet tested what happens if you pass in mismatched dask array types (eg: a cupy-backed image dask array, and a numpy-backed weights dask array). There might need to be a check & informative error message added here for that